### PR TITLE
Add Librtmp package

### PR DESCRIPTION
**Pull request by caseymcc**
Add Librtmp package

### CODE REVIEW

File: cmake/projects/librtmp/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# List of versions here...
+
+hunter_add_version(
+    PACKAGE_NAME
+    librtmp
+    VERSION
+    "2.4.0-p0"
+    URL
+    "https://github.com/caseymcc/rtmpdump/archive/2.4.0-p0.tar.gz"
```
Comment (ruslo) [inline cmake/projects/librtmp/hunter.cmake]: Please use this one instead:
* https://github.com/hunter-packages/rtmpdump

**Comment by ruslo** [comment]
> Add librtmp 5b2f14f

I still see unrelated "Submodule gate updated 0 files" change.

**Comment by caseymcc** [comment]
Should be gone now.

**Comment by ruslo** [comment]
> Should be gone now

Looks good, thank you. Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1297
* https://travis-ci.org/ingenue/hunter/builds/189112878

**Comment by ruslo** [comment]
> Testing

Some tests failed. Feel free to investigate fixes or simply exclude broken toolchains. Branch [pkg.librtmp](https://github.com/ingenue/hunter/tree/pkg.librtmp).

**Comment by caseymcc** [comment]
Fixed the android and VS12 issues, should have an update coming. Waiting on the MacOS test. VS11 and below will be turned off as they do not support C99 which is quasi used in the code. Also disabling analyze, mingw and msys as the builds seem to be failing from openssl and checking the openssl test those are excluded.

**Comment by caseymcc** [comment]
Updated hunter-packages/rtmpdump also with new release (used same name). Modified this PR with the new SHA and sent PR(https://github.com/ingenue/hunter/pull/46) for ingenue/hunter

**Comment by caseymcc** [comment]
Ok, this one should be a go, test updates in https://github.com/ingenue/hunter/pull/46.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1307
* https://travis-ci.org/ingenue/hunter/builds/191887004

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.1311
* https://travis-ci.org/ingenue/hunter/builds/191914925

**Comment by ruslo** [comment]
> https://travis-ci.org/ingenue/hunter/jobs/191914938

This job failed because `librtmp` is not relocatable in fact. Please take a look at `librtmpTargets.cmake` installed file - there should be no full paths there but OpenSSL and ZLIB is saved with full paths. This can be fixed by using `OpenSSL::*` targets and `ZLIB::zlib`. See wiki pages:
* https://github.com/ruslo/hunter/wiki/pkg.openssl
* https://github.com/ruslo/hunter/wiki/pkg.zlib

Wiki about relocation:
* https://github.com/ruslo/hunter/wiki/example.find_package.config

**Pull request merged**